src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/ProfilerEditingTarget.java

```diff
@@ -528,6 +524,17 @@ public String getDefaultNamePrefix()
       return "Profile";
    }
    
+   private void clearProfileCache()
+   {
+      server_.clearProfile(htmlLocalPath_, new ServerRequestCallback<JavaScriptObject>()
+      {
+         @Override
+         public void onError(ServerError error)
```

Comment:
Great, learning about logging now. Added a bunch of missing logs in data import and profiler.